code/__DEFINES/access.dm

```diff
@@ -58,9 +58,9 @@ GLOBAL_LIST_EMPTY(all_req_one_access)
 #define ACCESS_IFF_PMC				999
 //=================================================
 
-#define PAYGRADES_MARINE list("C","CMN","E1","E2","E3","E3E","E4","E5","E6","E7","E8","E8E","E9","E9E","O1","O2","O3","O4","O5","O6","WO","CWO","PO","CPO","MAJ")
+#define PAYGRADES_MARINE list("C","CMN","E1","E2","E3","E3E","E4","E5","E6","E7","E8","E8E","E9","E9E","O1","O2","O3","O4","O5","O6","WO","CWO","PO3","PO2","PO1","CPO","MAJ")
 #define PAYGRADES_OFFICER list("O1","O2","O3","O4","O5","O6","WO","CWO","CPO","MAJ")
-#define PAYGRADES_ENLISTED list("C","CMN","E1","E2","E3", "E3E","E4","E5","E6","E7","E8","E9","PO")
+#define PAYGRADES_ENLISTED list("C","CMN","E1","E2","E3", "E3E","E4","E5","E6","E7","E8","E9","PO3","PO2","PO1")
```

Comment:
Although I'm fairly certain it doesn't matter in the slightest. I'd like these ranks to be in order, not reverse order.

Reply:
Are these really PO3 > PO2 > PO1 ? what is going on with this military?

Reply:
Unironically yes, this is how the US Navy does it